server/routes/createReport.js

```diff
@@ -117,13 +124,21 @@ module.exports = function NewIncidentRoutes({ reportService, offenderService, in
     const involvedStaff =
       (input && input.involvedStaff) || (formId && (await involvedStaffService.getDraftInvolvedStaff(formId))) || []
 
+    const prison = await locationService.getPrisonById(token, agencyId).catch(err => {
+      logger.warn(err, 'Prison details not found in elite-2')
+      return { description: getIn(['assignedLivingUnit', 'agencyName'], offenderDetail) }
```

Comment:
similarly - don't think there is any point handling this.

Reply:
I do think this needs exception-handling because when the user enters the incident-details page for the first time and hasn't yet saved the page, there won't be a value for agencyId. So calling locationService.getPrisonById(token, prisonId) will cause an exception. 
And in this instance, we need to display the actual prison the prisoner is currently at via offenderService.getOffenderDetails(token, bookingId)